ngraph/test/util/engine/interpreter_engine.cpp

```diff
@@ -32,6 +35,45 @@ namespace
         return ngraph::test::all_close_f(expected, result, tolerance_bits);
     }
 
+    template <typename T>
+    typename std::enable_if<std::is_floating_point<T>::value, testing::AssertionResult>::type
+        compare_with_fp_tolerance(const std::shared_ptr<ngraph::op::Constant>& expected_results,
+                                  const std::shared_ptr<ngraph::runtime::Tensor>& results,
+                                  const T tolerance)
```

Comment:
Why do we need template tolerance?! You can use float

Reply:
Fixed.